community/modules/scripts/wait-for-startup/main.tf

```diff
@@ -13,22 +13,40 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+locals {
+  instance_names = compact(concat(var.instance_names, [var.instance_name]))
```

Comment:
You can not use `compact` here for the same reasons.

Reply:
Oops, sorry I missed that.  Fixed now.